community-operators/gitlab-runner-operator/1.1.0/manifests/gitlab-runner-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,536 @@
+                      - name: WATCH_NAMESPACES
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.annotations['olm.targetNamespaces']
+                    image: registry.gitlab.com/gitlab-runner-operator/gitlab-runner-operator:v1.1.0
+                    name: manager
+                    ports:
+                      - containerPort: 9443
+                        name: webhook-server
+                        protocol: TCP
+                    resources:
+                      limits:
+                        cpu: 100m
+                        memory: 140Mi
+                      requests:
+                        cpu: 100m
+                        memory: 120Mi
+                terminationGracePeriodSeconds: 10
+      permissions:
+        - rules:
+            - apiGroups:
+                - ""
+              resources:
+                - configmaps
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - ""
+              resources:
+                - configmaps/status
+              verbs:
+                - get
+                - update
+                - patch
+            - apiGroups:
+                - ""
+              resources:
+                - events
+              verbs:
+                - create
+                - patch
+          serviceAccountName: default
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: false
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  keywords:
+    - GitLab
+    - CI/CD
+    - DevOps
+    - SAST
+    - DAST
+  links:
+    - name: Gitlab Runner Operator
+      url: https://gitlab.com/gitlab-org/gl-openshift/gitlab-runner-operator
+    - name: GitLab Docs
+      url: https://docs.gitlab.com
+    - name: GitLab Runner
+      url: https://docs.gitlab.com/runner/
+    - name: GitLab CI/CD
+      url: https://docs.gitlab.com/ee/ci/quick_start/
+  maintainers:
+    - email: support@gitlab.com
+      name: Georgi N. Georgiev
+  maturity: stable
+  provider:
+    name: GitLab, Inc.
+  version: 1.1.0
+  webhookdefinitions:
+    - admissionReviewVersions:
+        - v1beta1
+      containerPort: 443
+      deploymentName: gitlab-runner-controller-manager
+      failurePolicy: Fail
+      generateName: vrunner.kb.io
+      rules:
+        - apiGroups:
+            - apps.gitlab.com
+          apiVersions:
+            - v1beta2
+          operations:
+            - CREATE
+            - UPDATE
+          resources:
+            - runners
+      sideEffects: None
+      targetPort: 9443
+      type: ValidatingAdmissionWebhook
+      webhookPath: /validate-apps-gitlab-com-v1beta2-runner
+  replaces: 13.10.0
```

Comment:
You would have to put correct replaces. In your case
```
replaces: gitlab-runner-operator.v1.1.0
```

Reply:
This is v1.1.0, shouldn't this be `replaces: gitlab-runner-operator.v13.10.0`?